source/features/toggle-everything-with-alt.tsx

```diff
@@ -27,7 +27,7 @@ function clickAllExcept(elementsToClick: string, except: HTMLElement): void {
 	}
 }
 
-function allDiffsSelector(): string {
+function diffsSelector(): string {
```

Comment:
I took the chance to make the function names consistent, `all` is redundant after all